### PR TITLE
send fulfilment request for all telephone capture

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Log fulfilment against telephone capture

# What has changed
Send fulfilment for all types of case now

# How to test?
test with AT and actionscheduler branches from this ticket

# Links
https://trello.com/c/fRwTHZam/483-rmc-335-log-telephone-capture-request-5

# Screenshots (if appropriate):

### CODE REVIEW

**Review by LeoKHoward — APPROVED** [review]
Ran action scheduler/case-api tests and acceptance tests and they all passed

File: src/main/java/uk/gov/ons/census/caseapisvc/endpoint/CaseEndpoint.java [hunk 1]
```diff
@@ -38,8 +38,9 @@
 @RestController
 @RequestMapping(value = "/cases")
 public final class CaseEndpoint {
-
   private static final Logger log = LoggerFactory.getLogger(CaseEndpoint.class);
+  private static final String RM_TELEPHONE_CAPTURE_HOUSEHOLD_INDIVIDUAL = "RM_TC_HI";
+  private static final String RM_TELEPHONE_CAPTURE_HOUSEHOLD = "RM_TC";
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/caseapisvc/endpoint/CaseEndpoint.java]: Won't always be a household telephone capture request.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
I'm getting IT's failing with `DataIntegrityViolationException` every time I try to build off this branch?

File: src/main/java/uk/gov/ons/census/caseapisvc/endpoint/CaseEndpoint.java (hunk 1 above)
Comment (LukeLoze1) [reply src/main/java/uk/gov/ons/census/caseapisvc/endpoint/CaseEndpoint.java]: changed

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**